tradeexecutor/state/statistics.py

```diff
@@ -69,23 +69,24 @@ class FinalPositionStatistics:
 @dataclass
 class PortfolioStatistics:
     #: Real-time clock when these stats were calculated
```

Comment:
Class comments need to be docstrings. See any other documented class in the project.

Reply:
Have changed class comments to docstrings